pkg/util/mount/nsenter_mount.go

```diff
@@ -170,6 +170,12 @@ func (n *NsenterMounter) IsLikelyNotMountPoint(file string) (bool, error) {
 		return true, err
 	}
 
+	// Check the directory exists
+	if _, err = os.Stat(file); os.IsNotExist(err) {
```

Comment:
Let's imagine we have a directory `/var/lib/kubelet/test` which is a mount point which contains directory `foo`. Now, I think that if we call `IsLikelyNotMountPoint` for it `var/lib/kubelet/test/foo` we'd get false for newer versions of `findmnt` (see comment in line 186). If that's the case (I haven't tested this) your change would introduce a semantic change as now we'd return true.

I'm not saying it's a bad change, but we should say it explicitly.